src/coreclr/nativeaot/Runtime/thread.h

```diff
@@ -112,6 +112,9 @@ struct ThreadBuffer
 #ifdef FEATURE_GC_STRESS
     uint32_t                m_uRand;                                // current per-thread random number
 #endif // FEATURE_GC_STRESS
+#if defined(TARGET_UNIX) && !HAVE_MACH_EXCEPTIONS && !defined(HOST_TVOS)
+    void *                   m_alternateStack;                      // ptr to alternate signal stack
```

Comment:
A nit - I've just noticed that the indentation is off by one space
```suggestion
    void *                  m_alternateStack;                      // ptr to alternate signal stack
```